components/apimgt/org.wso2.carbon.apimgt.internal.service/src/gen/java/org/wso2/carbon/apimgt/internal/service/SynapseArtifactsApiService.java

```diff
@@ -0,0 +1,24 @@
+package org.wso2.carbon.apimgt.internal.service;
+
+import org.wso2.carbon.apimgt.internal.service.*;
```

Comment:
Rather than using wild card import. use the exact classes.

Reply:
Fix this issue in other places/files as well.

Reply:
This is an auto-generated file. We don't need to modify this.